### PR TITLE
Upload DEI Poster

### PR DESCRIPTION
**Pull request by rayegun**
Someone was asking I think by email for more info.

So I just quick and dirty uploaded @skygering's poster and added Diversity Scholarships as a header. 

@lucaferranti @skygering

### CODE REVIEW

File: config.md [hunk 1]
```diff
@@ -152,6 +152,7 @@ configuration = Dict(
             "Venue" => "/2024/venue",
             "Volunteer" => "/2024/volunteer",
             "Code of Conduct" => "/2024/coc",
+            "Diversity Scholarships" => "2024/dei"
```
Comment (lucaferranti) [inline config.md:155]: The tab also has info about childcare, but that does not fit under diversity scholarship I believe?

1. Could rename the tab "Diversity, Equity and Inclusion" (although maybe the name too long for a title in a navigation bar)
2. move the childcare section e.g. under venue

**Review by lucaferranti — APPROVED** [review]
I added a few clarifications based on what skylar had dm'ed me (updated childcare age and more details about scholaship award.

File: config.md (hunk 1 above)
Comment (bvdmitri) [reply config.md:155]: What about "Diversity and Inclusion"?

Comment (lucaferranti) [reply config.md:155]:
```suggestion
            "Diversity and Inclusion" => "2024/dei"
```

File: config.md [hunk 2]
```diff
@@ -152,6 +152,7 @@ configuration = Dict(
             "Venue" => "/2024/venue",
             "Volunteer" => "/2024/volunteer",
             "Code of Conduct" => "/2024/coc",
+            "Diversity and Inclusion" => "2024/dei"
```
Comment (vchuravy) [inline config.md:155]:
```suggestion
            "Diversity and Inclusion" => "/2024/dei"
```

**Pull request merged**